src/Game/Scenes/GameSceneInputHandler.cs

```diff
@@ -52,11 +54,13 @@ partial class GameScene
         private Entity _queuedObject;
         private bool _rightMousePressed, _continueRunning;
 
-
         public bool IsMouseOverUI => Engine.UI.IsMouseOverAControl && !(Engine.UI.MouseOverControl is WorldViewport);
 
 	    private bool _isShiftDown;
 
+        [DllImport("user32.dll", CharSet = CharSet.Auto, ExactSpelling = true, CallingConvention = CallingConvention.Winapi)]
```

Comment:
it's not crossplatform!

Reply:
[DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
        [return: MarshalAs(UnmanagedType.Bool)]
        private static extern bool SetDllDirectory(string lpPathName);

not crossplatform too ?